source/nodejs/adaptivecards-designer-app/src/app.ts

```diff
@@ -61,9 +61,13 @@ window.onload = function() {
     }
     */
 
-	let designer = new ACDesigner.CardDesigner(ACDesigner.defaultMicrosoftHosts);
-	designer.sampleCatalogueUrl = window.location.origin + "/sample-catalogue.json";
-	designer.attachTo(document.getElementById("designerRootHost"));
+  let designer = new ACDesigner.CardDesigner(
```

Comment:
I can't see the change on https://adaptivecardsci.z5.web.core.windows.net/pr/7051/designer

Reply:
Strange, I can't either. I wasn't aware there was CI for this in this project. I can attempt to look at the action that ran to see if there was an issue.

Reply:
It looks like [an action to build and deploy this PR succeeded](https://microsoft.visualstudio.com/AdaptiveCards/_build/results?buildId=45740634&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=08d41a1b-6013-5b01-b962-661a2fb402c0&l=1694), but since I'm unfamiliar with this project's infrastructure details I can't know for sure why you wouldn't see this PR's changes.

Reply:
you may need to update adaptivecards\source\nodejs\adaptivecards-site\themes\adaptivecards\layout\designer.ejs too.
Your change is made to adaptivecards-designer-app, but the site doesn't use it

Reply:
I think I found the code that initializes the designer for the page in question and pushed the appropriate changes just now.

Reply:
The device emulation picker now appears at https://adaptivecardsci.z5.web.core.windows.net/pr/7051/designer